core/arch/arm/include/mm/core_mmu.h

```diff
@@ -122,9 +122,12 @@ enum teecore_memtypes {
 	MEM_AREA_MAXTYPE
 };
 
-/* Default NSec shared memory allocated from NSec world */
-extern unsigned long default_nsec_shm_paddr;
```

Comment:
This old name is more clear than the new, especially since nsec_shm or NSEC_SHM is used at many places.